src/sass/app_2.scss

```diff
@@ -7,5 +7,6 @@
 @import 'app_2/_common/typography.scss';
 @import 'app_2/_common/buttons.scss';
 @import 'app_2/_common/input_field.scss';
-@import 'app_2/_common/dropdown.scss';
+@import 'app_2/_common/select.scss';
```

Comment:
Probably needs this renamed back to `dropdown.scss`.

Reply:
@zakibinary done.